Merge section 2 and 2.1 (#1)
adonath/gammapy-v1.0-paper: src/ms-review.tex

  migration, $\mu=\frac{E}{E_{\rm true}}$.
\end{itemize}

In addition to the instrument characteristics described above, there is also the instrumental
background, that results from hadronic events being misclassified as \gammaray events. These

> events constitute a uniform background to the \gammaray events For \fermi the residual hadronic background

Reviewer: ```suggestion
events constitute a uniform background to the \gammaray events. For \fermi the residual hadronic background
```

> is very small ($<1\%$) because of its veto layer and can often be neglected. For IACTs and WCTs in contrast

Reviewer 2: ```suggestion
is very small ($<1\%$) because of its veto layer and can often be neglected. For IACTs and WCDs in contrast,
```

the background can account for a very large part ($>95\%$) of the events and must be treated
accordingly in the analysis. As the background is very specific to the instrument, \gammapy typically 
relies on the background models provided with the DL3 data. The background typically only depends

> on the reconstructed event positon and energy $Bkg(p, E)$. However in general all IRFs depend on the

Reviewer: ```suggestion
on the reconstructed event position and energy $Bkg(p, E)$. However in general all IRFs depend on the
```

defined by a binned energy axis and of spatial dimensions, which either define 
a spherical projection from celestial coordinates to a pixelised image space
or a single region on the sky.

After all data have been projected into the same geometry, it is typically
required to improve the residual hadronic background estimate. As residual hadronic
background models can be subject to significant systematic uncertainties,
these models can be improved by taking into account actual data
from regions without known \gammaray sources. This includes methods 
such as the ring or the FoV background techniques or background measurements
performed within, e.g. reflected regions~\citep{Berge07}.

Data measured at the FoV or energy boundaries of the instrument are typically
associated with a systematic uncertainty in the IRF. For this reason this part 
of the data is often excluded from subsequent analysis by defining regions of
\enquote{safe} data in the spatial as well as energy dimension.

All the code related to reduce the DL3 data to binned data structures 
is contained in the \code{gammapy.makers} sub-package.

>  \subsubsection{DL4: Binned Data Structures}

Reviewer: Why not making this a subsection. It is strange to have one subsubsection only

Author: Indeed this happened by mistake. Fixed.

 The counts data and the reduced IRFs in the form of maps are bundled into datasets
 that represent the fourth data level (DL4). These reduced datasets can be written to disk,
 in a format specific to \gammapy to allow users to read them back at any time later
 for modelling and fitting. Different variations of such datasets support different 
 analysis methods and fit statistics. The datasets can be used to perform a joint-likelihood
 fit, allowing one to combine different measurements, e.g. from different observations
 but also from different instruments or event classes. They can also be used for binned
 simulation as well as event sampling to simulate DL3 events data.

Binned maps and datasets, which represent a collection of binned
maps, are defined in the \code{gammapy.maps} and \code{gammapy.datasets}
sub-packages, respectively. 

\subsection{From DL4 to DL5/6: Modeling and Fitting}
The next step is then typically to model the datasets using binned Poisson maximum likelihood fitting.

> Assuming Poisson statistics per bin the log-likelihood of an observation is given by the Cash statistics\citep{Cash}:

Reviewer: Since there are two possible likelihoods, I would avoid giving the explicit Cash expression but simply mention 
$\log L(N_{obs, i}| N_{pred, i}$ and possibly cite Cash as an example.

Author: I see you point, but I think the Cash statistics is the more fundamental and important one. With Gammapy we have made the transition from the "classical methods" to more modern ones, which are typically based on background templates. The community will follow...

%
\begin{equation}

> 	\mathcal{C} = \sum_{i=0}^K N_{\mathrm{Pred, i}} - N_{\mathrm{Obs, i}} \log{N_{\mathrm{Pred, i}}} 

Reviewer: ```suggestion
	\mathcal{C} = 2 \sum_{i=0}^K N_{\mathrm{Pred, i}} - N_{\mathrm{Obs, i}} \log{N_{\mathrm{Pred, i}}} 
```

\end{equation}

Where the expected number of events in a \gammaray observation is given by:
 \begin{equation}
	 \begin{split}
		N_{\mathrm{Pred}}(p, E; \hat{\theta})\ {\rm d}p\ {\rm d}E =  &E_{\rm disp} \circledast \left[ PSF \circledast \left( A_{\rm eff} \cdot t_{\rm obs} \cdot \Phi(\hat{\theta}) \right) \right]\\
						& + Bkg(p, E) \cdot t_{\rm obs}
	 \end{split}
 \end{equation}

With the IRF components described in the previous section as well as an analytical model to describe the

> intensity of the radiation from gamma-ray sources as a function of the energy, $E_{\rm true}$,

Reviewer: this misses a verb no?

Author: Fixed

and of the position in the FoV, $p_{\rm true}$:
%
\begin{equation}
    \Phi(p_{\rm true}, E_{\rm true}; \hat{\theta}), [\Phi] = \si{TeV^{-1}.cm^{-2}.s^{-1}}
    \label{eq:general_source_model}
\end{equation}
Where $\hat{\theta}$ is a set of model parameters that can be adjusted in a fit. 

> Observtaions can be either modelled individually, or in a joint likelihood analysis.

Reviewer: ```suggestion
Observations can be either modeled individually, or in a joint likelihood analysis.
```

> In the latter case the total (jpint) log-likleihood is given by the sum of the log-likelihoods

Reviewer: ```suggestion
In the latter case the total (joint) log-likelihood is given by the sum of the log-likelihoods
```

per observation $\mathcal{L} = \sum{i=0}^M \mathcal{C_i}$ for M observations. Other fit
statistics such as \textit{WStat} or a simple $\chi^2$ for flux points are also often used
and are supported by \gammapy as well. Models can be simple analytical models or more complex
ones from radiation mechanisms of accelerated particle populations (e.g. inverse Compton or $\pi^{0}$ decay).

Independently or subsequently to the global modelling, the data can be re-grouped to compute
flux points, light curves and flux maps as well as significance maps in different energy bands.

> Parametric models and all the functionality realted to fitting is implemented in 

Reviewer: ```suggestion
Parametric models and all the functionality related to fitting is implemented in 
```
